### PR TITLE
Set the mirror I want in the config or on cli.

### PR DESCRIPTION
**Pull request by vindarel**
Hi !

pirate-get didn't work for me because I can not access the default url nor the mirrors source.

With this commit, one can set a working mirror in the config or as an argument. I can then use pirate-get with tpb.today.
Isn't that great ? :)

Cheers

### CODE REVIEW

File: pirate/pirate.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
 from os.path import expanduser, expandvars
 from pirate.print import Printer
 
+MIRROR_DEFAULT = 'https://thepiratebay.mn'
+MIRROR_SOURCE = 'https://proxybay.co/list.txt'
```
Comment (rnhmjoj) [inline pirate/pirate.py]: Can you move these into `pirate/data.py`?

File: pirate/pirate.py [hunk 2]
```diff
@@ -310,7 +343,9 @@ def pirate_main(args):
     if args.source == 'local_tpb':
         results = pirate.local.search(args.database, args.search)
     elif args.source == 'tpb':
-        results, site = search_mirrors(printer, args.pages, args.category, args.sort, args.action, args.search)
+        results, site = search_mirrors(printer, args.pages, args.category,\
```
Comment (rnhmjoj) [inline pirate/pirate.py]: There's no need to escape the newline.

**Comment by vikstrous** [comment]
Please update the tests.

**Comment by vindarel** [comment]
Hey, here are the tests ! I encountered two pbs:

* I couldn't run the whole test suite, it makes my pc have 100% cpu load for more than 5 minutes… I don't think it's normal, is it ?
* I couldn't see the circle-ci result because of a "request timeout while talking to circle-productions...". I also read "Your build has exceeded the memory limit of 4G on 1 container"

Any suggestions ? Regards.

**Comment by rnhmjoj** [comment]
Implemented in bba0f412242a5a389de4ae2b132a884c6afc046e

**Pull request closed without merging**